src/components/App.js

```diff
@@ -2145,9 +2145,14 @@ class App extends Component {
                     .map(x => x.trim())
                     .map(x => {
                         let match = x.match(/(pass|[A-Za-z]\d+)(\s+(pass|[A-Za-z]\d+))*$/)
-                        if (match == null) return null
-
-                        return [x.slice(0, match.index), match[0].split(/\s+/)]
+                        if (match == null)
+                          return null
+                        let match2 = match[0].match(/pass/)
```

Comment:
Can you give this variable a more meaningful name, like `matchPass`? We can rename `match` to `matchPV` or something.

Reply:
done